### DISCUSSION TITLE
Ideas for DevOps Roadmap

### DISCUSSION THREAD

**Discussion by kaiwalyakoparkar** [commclassroom/roadmaps#19] [Ideas]
Please add your ideas here

**Comment by KKK2504** [comment]
**Follow these steps for getting started in Devops**

1.Learn programming languages.
2.Study operating systems.
3.Review networking security and protocols.
4.Understand Infrastructure as Code.
5.Adopt Continuous Integration/Continuous Deployment tools.
6.Invest in application and infrastructure monitoring.
7.Study cloud providers.
8.Learn cloud design 

**Visit this blog to have more clear idea in devops roadmap**

[https://blog.hubspot.com/website/devops-roadmap](url)

**Comment by SD007CSE** [comment]
I would like to make a PR to add web server block(eg. NGNIX ) in the readme

**Comment by codesaurabhk** [comment]
How can i find mentorship

**Comment by kaiwalyakoparkar** [comment]
Adding my youtube videos and blogs
https://youtube.com/@kaiwalya_koparkar/
https://blogs.kaiwalyakoparkar.com/
